# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/dom/idlharness-shadowrealm.window.js.ini
# Commit: d27c32ae0908
# Full Hash: d27c32ae0908c0b5d2c9213b9a624e3d0b71e2d5
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2024-01-23 21:57:50
# Regressor Bug: 1830781
# File Overlap Count: 1
# Description:
#   Bug 1830781 - Implement AbortSignal.any() r=saschanaz,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D198011
# ==============================================================================

diff -r 8af7503ecb57 -r d27c32ae0908 testing/web-platform/meta/dom/idlharness-shadowrealm.window.js.ini
--- a/testing/web-platform/meta/dom/idlharness-shadowrealm.window.js.ini	Tue Jan 23 08:04:52 2024 +0000
+++ b/testing/web-platform/meta/dom/idlharness-shadowrealm.window.js.ini	Tue Jan 23 08:43:07 2024 +0000
@@ -140,6 +140,3 @@
 
   [AbortController interface: operation abort(optional any)]
     expected: FAIL
-
-  [AbortSignal interface: operation any(sequence<AbortSignal>)]
-    expected: FAIL